### DISCUSSION TITLE
Issue when running on Ubuntu server on Forge

### DISCUSSION THREAD

**Discussion by lokmanm** [spatie/lighthouse-php#34] [Q&A]
Hi everyone,

I am trying to run this on Forge but encountering this issue:

```bash
2023-10-18 14:42:51] production.INFO: Lighthouse Failed: Could not run Lighthouse. Error : `node:internal/process/promises:279
            triggerUncaughtException(err, true /* fromPromise */);
            ^

Error: connect ECONNREFUSED 127.0.0.1:45237
    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1278:16) {
  errno: -111,
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 45237
}
`  
```

I have chromium installed and `chrome-launcher`, and `lighthouse` but it still fails. Locally it works just fine.
I even tried to kill the `chromium` instances on our Ubuntu server but still the issue persists.

Tried with Node 18, then I tried downgrading to 16 but the issue is still present.

Does anyone have any valuable tips on how to solve this?

**Comment by lokmanm** [comment]
If anyone has the same issue, I was able to fix this by installing regular chrome and not chromium.

**Reply by bardolf69** [reply]
How did you install Chrome?  I'm facing the same issue as you had originally and I installed Chrome like so
```
wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb
dpkg -i google-chrome-stable_current_amd64.deb
apt -f install
```
but I'm still getting the same error.

**Comment by mysticseagull** [comment]
@freekmurze any ideas on a setup for these? this seems to be a consistent issue, even in regards to using it within laravel sail, I have yet to get it running, but I assume there is no documentation for it.
